app_dart/test/service/branch_service_test.dart

```diff
@@ -188,6 +189,8 @@ void main() {
         throw gh.GitHubError(MockGitHub(), 'Failed to list commits');
       };
       await branchService.branchFlutterRecipes(branch);
```

Comment:
Can we add an explicit regression test? In the current state, this could easily be removed as it's not obvious what it's testing.

Reply:
Added a new regression test, feel free to resolve!